demo/demo.py

```diff
@@ -15,11 +15,11 @@
 root.addHandler(ch)
 
 if __name__ == '__main__':
-    ldclient._api_key = 'api_key'
+    ldclient.api_key = 'api_key'
```

Comment:
I did a wholesale change- including removing the api_key prefix from the auth header.